bin/matrix-search.js

```diff
@@ -24,24 +24,25 @@ Matrix.localization.init(Matrix.localesFolder, Matrix.config.locale, function ()
 
     Matrix.firebase.app.search(needle, function (data) {
       Matrix.loader.stop();
-      if (!_.isNull(data)) {
-        debug(data);
 
-        if ( !_.isArray(data)){
-          data = [ data ];
+      debug(data);
+      if (data.meta.visible === false) {
```

Comment:
This will error if `data` is null. That's why we do the check at the beginning. Please restore the isNull check.